src/bilder/images/dagster/deploy.py

```diff
@@ -267,11 +268,17 @@
 for product in hashicorp_products:
     configure_hashicorp_product(product)
 
+# Install and configure vector
+vector_config = VectorConfig(is_docker=True, use_global_log_sink=True)
+vector_config.configuration_templates[
+    TEMPLATES_DIRECTORY.joinpath("vector", "dagster_logs.yaml")
+] = {}
+install_vector(vector_config)
+configure_vector(vector_config)
```

Comment:
Should we wrap all of this as a single deploy function to cut down on boilerplate? (not right now, but as a TODO)